react-components/toolbar-filter/assets/table.js

```diff
@@ -16,7 +16,7 @@ import {
 } from '@patternfly/react-core';
 import { SearchIcon } from '@patternfly/react-icons';
 
-class AppTable extends React.Component {
+class DemoTable extends React.Component {
```

Comment:
```suggestion
class `DemoTable` extends `React.Component {`
```

Reply:
this is actual JS code that I need to run. I cannot add the ticks.